app/src/main/res/values/attrs.xml

```diff
@@ -15,6 +15,7 @@
     <attr name="tabRing" format="reference" />
     <attr name="snackbar" format="reference" />
     <attr name="foundation" format="reference" />
+    <attr name="foundationContrast" format="reference" />
```

Comment:
Can you add a description for this attribute similar to what we did for `menuItemButtonTintColor`? This is to help preserve some knowledge about these color variables since there's no particular person that fully owns/understands all the different color palette attributes anymore. It will also be helpful to add a description now about their use case and where they are used so that it will be helpful when we map these outs in the UI for the UX.